### PR TITLE
HW-01 ready

### PR DESCRIPTION
**Pull request by YaroslavKuzero**

### CODE REVIEW

File: contacts.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    const id = contactsList.length ? [...contactsList].pop().id + 1 : 1;
+    const newContact = {
+      id,
+      name,
+      email,
+      phone
+    };
+    contactsList.push(newContact);
+
+    fs.writeFile(this.contactsPath, JSON.stringify(contactsList))
```
Comment (odykyi) [inline contacts.js]: await

**Review by odykyi — CHANGES_REQUESTED** [review]
comments

File: contacts.js (hunk 1 above)
Comment (YaroslavKuzero) [reply contacts.js]: Changed.

**Comment by YaroslavKuzero** [comment]
> comments

Fixed

**Review by odykyi — APPROVED** [review]

**Pull request merged**